pdr_backend/lake/fetch_ohlcv.py

```diff
@@ -85,27 +83,52 @@ def safe_fetch_ohlcv_dydx(
     try:
         if exch != "dydx":
             return None
-        sinceIso = since.to_iso_timestr()
         headers = {"Accept": "application/json"}
         response = requests.get(
-            f"https://indexer.dydx.trade/v4/candles/perpetualMarkets/{symbol}"
-            + f"?resolution={timeframe}&fromISO={sinceIso}&limit={limit}",
-            headers=headers,
-            timeout=20,
-        )
-        response = requests.get(
-            "https://indexer.dydx.trade/v4/candles/perpetualMarkets/BTC-USD"
-            + "?resolution=5MINS&fromISO=2024-02-27T00:00:00.000Z&limit=1",
+            f"{BASE_URL_DYDX}/{symbol}?resolution={timeframe}"
+            f"&fromISO={since.to_iso_timestr()}&limit={limit}",
             headers=headers,
```

Comment:
The try/except loop should only be around the call to requests.get(), not to most of the method. Why: in general you want try/excepts to be as small and surgical as possible. Otherwise it gets really hard to debug.